onnxscript/rewriter/ort_fusions/attention.py

```diff
@@ -99,11 +108,11 @@ def pattern(
                 query_BSD,
                 key_BSD,
                 value_BSD,
-                # bias
-                # key_padding_mask
-                # attention_bias,
-                # past_key
-                # past_value
+                qkv_bias,
+                None,
```

Comment:
nit: adding `# key_padding_mask` or whatever this parameter corresponds to will help the reader.